tests/integration/exec.bats

```diff
@@ -57,3 +57,42 @@ function teardown() {
   [[ ${lines[1]} == *"."* ]]
   [[ ${lines[2]} == *".."* ]]
 }
+
+@test "runc exec ls -la with --cwd" {
+  # run busybox detached
+  runc run -d --console /dev/pts/ptmx test_busybox
+  [ "$status" -eq 0 ]
+
+  wait_for_container 15 1 test_busybox
+
+  runc exec --cwd /bin test_busybox pwd
+  [ "$status" -eq 0 ]
+
+  [[ ${output} == *"/bin"* ]]
```

Comment:
thanks @hqhq . why the output has a '\r' at the end? If we have no way to delete this character. A simpler way of doing it may be:
```
[[ ${output} == $(echo -e "/bin\r") ]]
```

Update patch done.